recipients.go

```diff
@@ -0,0 +1,70 @@
+package updown
+
+import (
+	"fmt"
+	"net/http"
+)
+
+// Recipients represents a end point type called by Updown on any event
+type Recipient struct {
+	ID    string `json:"id,omitempty"`
```

Comment:
same here, for consistency purposes, could we please capitalize the first letter only?